app/src/pages/p/[projectSlug]/usage/index.tsx

```diff
@@ -16,72 +16,240 @@ import {
+        <Tabs w="full">
+          <HStack justifyContent="space-between" marginX={0}>
+            <TabList>
+              <Tab>
+                <Heading size="md">Cost</Heading>
+              </Tab>
+              <Tab>
+                <Heading size="md">Activity</Heading>
+              </Tab>
+            </TabList>
+            <Spacer />
+            <Card w={"180px"} colorScheme="white" color={"gray.900"} padding={"5px"}>
+              <HStack spacing={0} justifyContent="space-between">
+                <Icon
+                  onClick={handlePrevMonthChange}
+                  cursor={"pointer"}
+                  as={ChevronLeftIcon}
+                  boxSize={5}
+                  color={"gray.900"}
+                />
+                <Text onClick={handlePrevMonthChange} cursor={"pointer"}>
+                  {formatToUTCDayMonth(startDate || startOfThisMonth)}
+                </Text>
+                <Text paddingX={"5px"}> - </Text>
+                <Text onClick={handleNextMonthChange} cursor={"pointer"}>
+                  {formatToUTCDayMonth(endDate || endOfThisMonth)}
+                </Text>
+                <Icon
+                  onClick={handleNextMonthChange}
+                  cursor={"pointer"}
+                  as={ChevronRightIcon}
+                  boxSize={5}
+                  color={nextMonthIsClickable() ? "gray.900" : "gray.400"}
+                />
+              </HStack>
             </Card>
-            <VStack spacing="4" width="300px" align="stretch">
-              <Card>
-                <CardBody>
-                  <Stat>
-                    <HStack>
-                      <StatLabel flex={1}>Total Requests</StatLabel>
-                      <Icon as={Hash} boxSize={4} color="gray.500" />
-                    </HStack>
-                    <StatNumber>
-                      {stats.data?.totals?.numQueries
-                        ? parseInt(stats.data?.totals?.numQueries.toString())?.toLocaleString()
-                        : undefined}
-                    </StatNumber>
-                  </Stat>
-                </CardBody>
-              </Card>
-              <Card>
-                <CardBody>
-                  <Stat>
-                    <HStack>
-                      <StatLabel flex={1}>Total Spent</StatLabel>
-                      <Icon as={DollarSign} boxSize={4} color="gray.500" />
-                    </HStack>
-                    <StatNumber>
-                      ${parseFloat(stats.data?.totals?.cost?.toString() ?? "0").toLocaleString()}
-                    </StatNumber>
-                  </Stat>
-                </CardBody>
-              </Card>
-            </VStack>
           </HStack>
+
+          <TabPanels>
+            <TabPanel paddingX={0}>
+              <HStack gap={4} align="start">
+                <Card flex={1}>
+                  <CardBody>
+                    <CostGraph
+                      startDate={startDate || startOfThisMonth}
```

Comment:
These default values for `startDate` and `endDate` should be unnecessary if we set default values for the query parameter properly, since it will always be defined.